### PR TITLE
imxrt106x: add BOARD_CONFIG to CFLAGS

### PR DESCRIPTION
**Pull request by HBuczynski**

### CODE REVIEW

**Comment by nalajcie** [comment]
please move this to the main Makefile, as the current approach is misleading (the `multi/imxrt-multi/Makefile` is included in the main Makefile so the `CFLAGS` will actually be changed for all devices compilation).

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,7 @@
 SIL ?= @
 MAKEFLAGS += --no-print-directory
 
+
```
Comment (nalajcie) [inline Makefile:12]: why? :P

**Review by nalajcie — APPROVED** [review]

**Comment by nalajcie** [comment]
As this is a not-destructive change, I'm merging this right away

File: Makefile (hunk 1 above)
Comment (HBuczynski) [reply Makefile:12]: I moved `CFLAGS` below line `include ../phoenix-rtos-build/Makefile.common`
Currently, in Makefile.common `CFLAGS` are reset.

This new line is a residue :P

**Pull request merged**